packages/components/organisms/f-content-cards/src/components/ContentCards.js

```diff
@@ -35,10 +37,33 @@ export default {
     data: () => ({
         state: STATE_LOADING,
         cards: [],
-        errors: []
+        errors: [],
+        cardObserver: {}
     }),
 
+    watch: {
+        errors (errors) {
+            if (errors.filter(e => e === STATE_ERROR).length === this.adapters.length) {
+                this.state = STATE_ERROR;
```

Comment:
So it looks like these states are only set when all adapters are erroring or returning no cards, is that right?

Reply:
yes that is correct as we would still want it to display cards if one of the adapters has succeeded